src/v2/Components/v2/ArtworkFilter/ArtworkFilters/ArtistsFilter.tsx

```diff
@@ -1,28 +1,69 @@
-import { Checkbox, Flex, Text, Toggle } from "@artsy/palette"
+import { Checkbox, Flex, Toggle } from "@artsy/palette"
 import { sortBy } from "lodash"
 import React, { FC, useEffect, useState } from "react"
-import styled from "styled-components"
 
 import { useArtworkFilterContext } from "../ArtworkFilterContext"
 import {
   FollowedArtistList,
   fetchFollowedArtists,
 } from "../Utils/fetchFollowedArtists"
 import { OptionText } from "./OptionText"
-
-const ToggleLink = styled(Text)`
-  cursor: pointer;
-  text-decoration: underline;
-`
-
-const INITIAL_ARTISTS_TO_SHOW = 6
+import { ShowMore } from "./ShowMore"
 
 interface ArtistsFilterProps {
   relayEnvironment?: any
   fairID?: string
   user?: User
 }
 
+const ArtistItem: React.FC<{
+  slug: string
+  name: string
+  followedArtistSlugs: string[]
+  isFollowedArtistCheckboxSelected: boolean
+}> = ({
+  slug,
+  name,
+  followedArtistSlugs,
+  isFollowedArtistCheckboxSelected,
+}) => {
+  const { currentlySelectedFilters, setFilter } = useArtworkFilterContext()
+  const toggleArtistSelection = (selected, slug) => {
+    let artistIDs = currentlySelectedFilters().artistIDs.slice()
+    if (selected) {
+      artistIDs.push(slug)
+    } else {
+      // When an artist is de-selected, if it is a followed artist _and_ that filter
+      // is also checked, we want to de-select it as well, and move remaining followed
+      // artists to the explicit `artistIDs` list.
+      artistIDs = artistIDs.filter(item => item !== slug)
+      if (followedArtistSlugs.includes(slug)) {
+        setFilter("includeArtworksByFollowedArtists", false)
+        artistIDs = artistIDs.concat(followedArtistSlugs)
+        artistIDs = artistIDs.filter(item => item !== slug)
+      }
+    }
+    setFilter("artistIDs", artistIDs)
+  }
+
+  const isFollowedArtist = followedArtistSlugs.includes(slug)
+  const selected =
+    currentlySelectedFilters().artistIDs.includes(slug) ||
+    (isFollowedArtistCheckboxSelected && isFollowedArtist)
+  const props = {
+    onSelect: selected => {
+      toggleArtistSelection(selected, slug)
+    },
+    selected,
+  }
+
+  return (
+    <Checkbox {...props}>
```

Comment:
I will keep commenting on these, every time. ***Use JSX***

Reply:
I moved this verbatim from elsewhere in this file, (altho I likely introduced the non-JSX props style originally 😅 )